plugins/woocommerce-blocks/assets/js/blocks/product-filters/edit.tsx

```diff
@@ -30,36 +30,36 @@ const TEMPLATE: InnerBlockTemplate[] = [
 		'woocommerce/product-filter',
 		{
 			filterType: 'active-filters',
-			heading: __( 'Active', 'woocommerce' ),
+			heading: 'Active',
 		},
 	],
 	[
 		'woocommerce/product-filter',
 		{
 			filterType: 'price-filter',
-			heading: __( 'Price', 'woocommerce' ),
+			heading: 'Filter by Price',
```

Comment:
Can you explain why do we change this? The current value matches the new design. Also, this heading, active and stock ones miss the translation wrapper function.

Reply:
Probably copy-pasted from one of the individual blocks, but I just fixed in https://github.com/woocommerce/woocommerce/pull/48638/commits/d9a31412f4ecdefa022d9d2144a53eee94555ad6